### ISSUE TITLE
Brew install php56-mysqlnd_ms fails

### ISSUE THREAD

**Issue by byCedric** [Homebrew/homebrew-php#1446]
I tried to install the php56-mysqlnd_ms formula, but it keeps failing. I tried all steps at https://github.com/Homebrew/homebrew-php#common-issues and https://github.com/Homebrew/homebrew-php#common-upgrade-issues.
I also tried to (un)tap homebrew-php and ran ``brew doctor``, ``brew cleanup``, ``brew cleanup -s``, ``rm -rf brew --cache``, ``brew update``, ``brew upgrade`` & ``brew prune``.
Looks like I can't get it to work...

Sidenote, this may be a duplicate of https://github.com/Homebrew/homebrew-php/issues/1365, but because of the lack of correct bug reporting I am not sure of that...

Parameter | Value
------------------ | ------------------
**OS X Version:** | 10.10.1
**Homebrew Version:** | Homebrew 0.9.5
**PHP Version in use:** | homebrew-php (5.6.3)
**Xcode Version:** | 6.1 (6A1052d)
**Output of gcc -v:** | Configured with: --prefix=/Library/Developer/CommandLineTools/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
 | Apple LLVM version 6.0 (clang-600.0.54) (based on LLVM 3.5svn)
 | Target: x86_64-apple-darwin14.0.0
 | Thread model: posix
**Output of php -v:** | PHP 5.6.3 (cli) (built: Nov 17 2014 13:35:29) 
 | Copyright (c) 1997-2014 The PHP Group
 | Zend Engine v2.6.0, Copyright (c) 1998-2014 Zend Technologies
**Output of brew install -v php56-mysqlnd_ms within a gist** | https://gist.github.com/byCedric/204e28be3e4583802236
**Output of brew doctor -v within a gist** | https://gist.github.com/byCedric/16894f959e813468e1cd

**Comment by alanthing** [comment]
Could you try uninstalling (maybe with `--force`?) all php-related packages, `brew update ; brew upgrade`, and try again? There have been a lot of changes in recent weeks. I'm closing due to the age of the issue, but re-open if your issue is not resolved.

**Comment by vojtech-dobes** [comment]
I removed all php56 packages except php56 itself, but installation of this one still fails:

```
$ brew install php56-mysqlnd_ms
==> Installing php56-mysqlnd_ms from josegonzalez/homebrew-php
==> Downloading http://pecl.php.net/get/mysqlnd_ms-1.4.2.tgz
######################################################################## 100.0%
==> /usr/local/opt/php56/bin/phpize
==> ./configure --prefix=/usr/local/Cellar/php56-mysqlnd_ms/1.4.2 --with-php-config=/usr/local/opt/php56/bin/php-config --enable-mysqlnd-ms
==> make
#               define PHPAPI __attribute__ ((visibility("default")))
                              ^
4 errors generated.
make: *** [php_mysqlnd_ms.lo] Error 1
 clang -I. -I/private/tmp/php56-mysqlnd_ms-cjfH7Q/mysqlnd_ms-1.4.2 -DPHP_ATOM_INC -I/private/tmp/php56-mysqlnd_ms-cjfH7Q/mysqlnd_ms-1.4.2/include -I/private/tmp/php56-mysqlnd_ms-cjfH7Q/mysqlnd_ms-1.4.2/main -I/private/tmp/php56-mysqlnd_ms-cjfH7Q/mysqlnd_ms-1.4.2 -I/usr/local/Cellar/php56/5.6.6/include/php -I/usr/local/Cellar/php56/5.6.6/include/php/main -I/usr/local/Cellar/php56/5.6.6/include/php/TSRM -I/usr/local/Cellar/php56/5.6.6/include/php/Zend -I/usr/local/Cellar/php56/5.6.6/include/php/ext -I/usr/local/Cellar/php56/5.6.6/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /private/tmp/php56-mysqlnd_ms-cjfH7Q/mysqlnd_ms-1.4.2/mysqlnd_ms_filter_user.c  -fno-common -DPIC -o .libs/mysqlnd_ms_filter_user.o

READ THIS: http://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/homebrew):
  https://github.com/josegonzalez/homebrew-php/issues
```

```
$ brew doctor
Your system is ready to brew.
```

**Comment by alanthing** [comment]
I've updated the formula and am waiting on the brew test bot to test it. I'll report back here when it's completed.

**Comment by vojtech-dobes** [comment]
Thanks!

**Comment by alanthing** [comment]
Looks like there's a known issue with mysqlnd_ms and PHP 5.6: https://bugs.php.net/bug.php?id=67113. I'm working on finding a patch.

**Comment by alanthing** [comment]
Patch pending brew test bot.

**Comment by alanthing** [comment]
Pushed in #1573; would love feedback on using it since all I did for the patch was verify that it builds without error.
